### ISSUE TITLE
New Circuit Dialog : Not working Properly

### ISSUE THREAD

**Issue by Shivansh2407** [CircuitVerse/CircuitVerse#1933]
**Describe the bug**
Whenever the user clicks on New Circuit, A dialog box appears asking for the name of the circuit, but if we click on cancel it generates an Untitled-Circuit.

**To Reproduce**
1. Go to 'beta.circuitverse.org'
2. Click on 'Simulator'
3. Click On 'New Circuit' -> Click Cancel

**Expected behavior**
It should not generate the Circuit, if the user clicks on Cancel. Not even the Untitled One.

**Device Information [optional]:**
 - OS: [Windows]
 - Browser [Chrome]

**Are you working on this issue? (Yes/No)**
No.

**Comment by Preeti2095** [comment]
i want to work on this issue.

**Comment by harshitaaaaaa** [comment]
This error is not occuring in my browser.

**Comment by Shivansh2407** [comment]
> This error is not occuring in my browser.

Did you use beta.circuitverse.org? Please specify your Device Info

**Comment by Shivansh2407** [comment]
> i want to work on this issue.

Go ahead @Preeti2095 👍

**Comment by Preeti2095** [comment]
when we open the simulator it gives us a default circuit page. it is designed like that. and when we try to delete that last tab it says there must me 2 tabs to delete it. that means we can't have a page with no tab. so, when you did the things which you have written for recreating the behavior for error, it does nothing and the default untitled page remains. so i wanted to ask can we remove that last tab also or developer has designed that way only???

**Comment by Shivansh2407** [comment]
So the thing is, We need to have atleast One Circuit to continue our work. We cannot remove the circuit if we have only one circuit left. The issue here is that when we click on new circuit, and click on cancel in the dialog box, it adds a new circuit named Untitled Circuit, which is unnecessary and needs to be fixed.

**Comment by satu0king** [comment]
@Shivansh2407 I think this is good if it renames as untitled. User can delete anyway ..

**Comment by Preeti2095** [comment]
> So the thing is, We need to have atleast One Circuit to continue our work. We cannot remove the circuit if we have only one circuit left. The issue here is that when we click on new circuit, and click on cancel in the dialog box, it adds a new circuit named Untitled Circuit, which is unnecessary and needs to be fixed.

For this I want to make clear that if I am understanding wrong or you are saying something else(Don't get offended):-
1) you launched the simulator
2) it will give you a circuit page with name untitled
3) so when you create new circuit but you canceled it
4) so now it shows you the circuit page named "untitled" which was already present when you first launched the simulator 
5) it doesn't create any new "untitled" page
6) and also you can change the name "untitled"
So in short, I think it is fine....means no issue!!

**Comment by Preeti2095** [comment]
Now I am not handling this issue.

**Comment by ZadeAbhishek** [comment]
i know this not big issue but i know the solution for this can i solve this.
